prettier.novaextension/extension.json

```
@@ -54,204 +58,241 @@
 			"children": [
 				{
 					"key": "prettier.module.path",
-					"title": "Prettier module",
-					"description": "Prettier module folder. If none is specified, it defaults to the Prettier installed in the project or, if there is none, the version embedded in the extension. Can be overridden in project settings.",
+					"title": "Prettier Module",
+					"description": "Set a folder for the Prettier module. If not set, it defaults to the build-in Prettier or the module in the project.",
```

```suggestion
					"description": "Set a folder for the Prettier module. If not set it defaults to Prettier installed in the project, or the built-in Prettier.",
```

I think the order is important as the project Prettier is preferred over the built-in one.